frontend/src/components/file-explorer/FileExplorer.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { waitFor, screen } from "@testing-library/react";
+import { waitFor } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { act } from "react-dom/test-utils";
```

Comment:
```suggestion
import { waitFor, act } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
```

This should fix: `Warning: ReactDOMTestUtils.act is deprecated in favor of React.act. Import act from react instead of react-dom/test-utils. See https://react.dev/warnings/react-dom-test-utils for more info.`